### PR TITLE
Initial UI

### PR DESCRIPTION
**Pull request by D-Antonelli**

### CODE REVIEW

File: src/requests/translations/index.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+export async function fetchTranslations(): Promise<SetStateAction<TranslationModel[]>> {
+  const request = "http://localhost:3001/v1/strings";
+
+  const response = await axios.get(request);
+  const responseData = response.data;
+  return responseData;
+}
+
+export async function patchTranslations(stringId:string, data: Update) {
+  const url = `http://localhost:3001/v1/strings/${stringId}/update`;
```
Comment (henryallsuch) [inline src/requests/translations/index.ts:16]: Can this API url be from an environment variable?

File: src/requests/translations/index.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  const result = await axios.patch(url, JSON.stringify(data), {headers: {"Content-Type": "application/json"}});
+  return result;
+}
+
+export async function postTranslationSettings(data: LanguageSettings) {
+  const url = "http://localhost:3001/v1/strings/import";
```
Comment (henryallsuch) [inline src/requests/translations/index.ts:22]: Can this API url be from an environment variable? Same here (maybe just the host & port)

File: src/components/languageDropdown.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  async function sendForm(e: FormEvent<HTMLFormElement>) {
+    e.preventDefault();
+    try {
+      if (sourceLanguage && targetLanguage) {
+        const data = new LanguageSettings(sourceLanguage, targetLanguage);
+        const response = await postTranslationSettings(data);
+        handleData(response.data);
+      }
+    } catch (error: any) {
+      console.log(error.message);
```
Comment (henryallsuch) [inline src/components/languageDropdown.tsx:36]: Could we show a message at this point?

File: src/components/localizeForm.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      setTranslations(values);
+    }
+  }
+
+  async function saveUpdate(stringId: Key, localization: Localization) {
+    try {
+      const update = new UpdateClass(localization._id.toString(), localization.text);
+      await patchTranslations(stringId.toString(), update);
+    } catch (error: any) {
+      console.log(error.message);
```
Comment (henryallsuch) [inline src/components/localizeForm.tsx:33]: Could we show just a simple error message? Maybe a generic one for now.

**Review by henryallsuch — CHANGES_REQUESTED** [review]
Nice work! It's a great start, just a few changes.

File: src/requests/translations/index.ts (hunk 2 above)
Comment (D-Antonelli) [reply src/requests/translations/index.ts:22]: 👍🏽

**Review by henryallsuch — APPROVED** [review]
👍  nice work

**Pull request merged**